### ISSUE TITLE
does the dataset include  camera pose information?

### ISSUE THREAD

**Issue by Wanghy118** [layumi/University1652-Baseline#31]
hi 
thanks for your excellent work！ I wonder does the dataset include camera pose information?

**Comment by layumi** [comment]
Yes. You need to read it from the xml file  https://github.com/layumi/University1652-Baseline#news

![image](https://user-images.githubusercontent.com/8390471/185731599-4ff361e1-7c57-464d-983d-bfdc2f48da5b.png)

**Comment by Wanghy118** [comment]
Thanks for your guidance!   I  have checked the value in the Tour_****.kml file.
However, I thought that the values of latitude, longitude, and altitude in the Tour_****.kml file represent the coordinate of the buildings, for the values do not change at all and are equal to the value in the ****.kml in the first-key folder.

For example,  in the tour_0020.kml, the  latitude, longitude, and altitude  with time are respectively:
![image](https://user-images.githubusercontent.com/61537293/186054710-7307b7f7-9b21-498a-bc51-1efdb02744c6.png)

So, I am still confused about whether the dataset includes **camera location** information. Please give me another clue if i did not fully understand your answer. Thank you！
